contracts/OwnedSet.sol

```diff
@@ -174,6 +176,19 @@ contract OwnedSet is Owned, ValidatorSet {
 		return pending;
 	}
 
+	// INTERNAL
+
+	// Called when an initiated change reaches finality and is activated.
+	// This method is defined with no modifiers so it can be reused by
+	// contracts inheriting it (e.g. for mocking in tests).
+	function finalizeChangeInternal()
+		internal
```

Comment:
Should we maybe move `!finalized` modifier to that function and only add system-account requirement in subcontracts?